### ISSUE TITLE
Permission error when creating triggers

### ISSUE THREAD

**Issue by tom2strobl** [neondatabase/neon#2230]
Hi there – I heard about neon from a hasura newsletter and am pretty stoked!

We switched a simple dev db to neon and all our _previous_ functions and triggers seemed to be imported nicely (although so far we only confirmed their existence and haven't run any tests yet). When we try to create new event triggers through the hasura console (which also makes sure we cannot have any syntax errors of any sort), we run into the following error (the actual SQL statement that was fired is within the json error respone):

```json
{
  "code": "unexpected",
  "error": "database query error",
  "internal": {
    "arguments": [],
    "error": {
      "description": null,
      "exec_status": "FatalError",
      "hint": null,
      "message": "permission denied for table project",
      "status_code": "42501"
    },
    "prepared": false,
    "statement": "\n CREATE TRIGGER \"notify_hasura_projectChange_UPDATE\" AFTER UPDATE ON \"public\".\"project\" FOR EACH ROW EXECUTE PROCEDURE hdb_catalog.\"notify_hasura_projectChange_UPDATE\"()\n "
  },
  "path": "$.args[0].args"
}
```

Note that the connection by hasura to the db is made with the first generated neon-user (what neon suggests).
Do we have to grant that user certain privileges somehow?

**Comment by kelvich** [comment]
Hi! Hm, seems that you are stepping on one of the problems with our current setup with user roles. Any chance that you've done initial schema creation through psql web link login or through SQL editor in the console? Right now, those connections are made using a different Postgres role called `web_access.` And some of the grants are missing between `web_access` and your user -- that is on us to fix.

As a workaround, you can try to create everything under your user (same username as GitHub login) -- that should work fine.

**Comment by tom2strobl** [comment]
Ah I see. Thanks for the swift response! Yes, I did what the first screen told me to do: https://neon.tech/docs/how-to-guides/import-an-existing-database/ (which is using the web login).

I can confirm after deleting and recreating the database in neon (for cleanliness sake) I was able to manually dump/restore not using the web login, but the connection string from the neon admin UI without issues and creating triggers from within hasura now works like a charm!

Simple example for anyone coming to this thread (-Fd is necessary since neon doesn't like plain-text, which kinda makes sense):

```bash
pg_dump <old-connection-string> -f db.dump -Fd
pg_restore db.dump -d <neon-connection-string>
```
